scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/policy/ram/RamSchedulingPolicy.java

```diff
@@ -0,0 +1,72 @@
+package org.ow2.proactive.scheduler.policy.ram;
+
+import org.apache.log4j.Logger;
+import org.objectweb.proactive.ActiveObjectCreationException;
+import org.objectweb.proactive.api.PAActiveObject;
+import org.objectweb.proactive.core.node.Node;
+import org.objectweb.proactive.core.node.NodeException;
+import org.ow2.proactive.scheduler.descriptor.EligibleTaskDescriptor;
+import org.ow2.proactive.scheduler.policy.ExtendedSchedulerPolicy;
+import org.ow2.proactive.utils.NodeSet;
+
+
+/**
+ * 
+ * This Policy is designed to handle preallocation of RAM into a node machine. 
+ * When the task contains the generic information ALLOC_RAM_GIGABYTES, 
+ * this policy will return false if there is not enough RAM available 
+ * or true if there is RAM available (in this last case it will set the ALLOC_RAM_GIGABYTES property at node level to book the RAM in the node machine)
+ *
+ */
+public class RamSchedulingPolicy extends ExtendedSchedulerPolicy {
+
+    private static final Logger logger = Logger.getLogger(RamSchedulingPolicy.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename the "logger" logger to comply with the format "LOG(?:GER)?". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1312)